### ISSUE TITLE
a11y: Color contrast of Clear all link in search results filters insufficient

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#143]
## Description:
Clear all links for filters on the search results page do not quite meet AA color contrast requirements (4.45:1).

## Screenshot:
![Screen Shot of color contrast calculation of clear all link showing 4.45:1 contrast ratio](https://user-images.githubusercontent.com/1093423/105542410-87f95600-5cb6-11eb-84d8-258272a3f291.png)

## WCAG SC
[Success Criterion 1.4.3 Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:

- **Large Text**
  - Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
- **Incidental**
  - Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
- **Logotypes**
  - Text that is part of a logo or brand name has no contrast requirement.

## Recommended fix:
Set `clear-all` class to use `color:#d34141;` to achieve 4.55:1 ratio in the CSS ([/app/assets/stylesheets/application.scss#L143](https://github.com/18F/all_sorns/blob/f8d5771a852dc60aa74caf84cc07b69b5cfb1ae1/app/assets/stylesheets/application.scss#L143))

**Comment by lauraGgit** [comment]
closed by #142
